app/code/Magento/Backend/Test/Mftf/ActionGroup/AdminOpenConfigurationStoresPageActionGroup.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminOpenConfigurationStoresPageActionGroup">
+        <annotations>
+            <description>Open configuration stores page.</description>
+        </annotations>
+
+        <amOnPage url="{{ConfigurationStoresPage.url}}" stepKey="goToConfigurationStoresPage"/>
```

Comment:
Could you also deprecate the old `Page` object and introduce new one that actually follows the Magento Best Practices (`AdminConfigurationStoresPage`)?

Reply:
@lbajsarowicz  Done